### ISSUE TITLE
b-dropdown-item-button has inconsistent styling with b-dropdown-item in Chromium browsers

### ISSUE THREAD

**Issue by bbugh** [bootstrap-vue/bootstrap-vue#5943]
### Describe the bug

In Chromium, `b-dropdown-item-button` shows a border when clicked (on `:focus`), but `b-dropdown-item` doesn't.

<img width="421" alt="image" src="https://user-images.githubusercontent.com/438465/96776425-a34e7f80-13ae-11eb-8d57-3da77700e39b.png">

(Backgrounds are lightened to show the border effect)

`b-dropdown-item-button` seems to be picking up the `button` `outline: -webkit-focus-ring-color` effect from Reboot. Firefox does not support `-webkit-focus-ring-color` so it show the single dotted outline.

https://github.com/twbs/bootstrap/blob/ebd9eb105ab094af747c09082ced1df71aaec123/scss/_reboot.scss#L419-L422

### Steps to reproduce the bug

1. Use a `b-dropdown-item-button`
2. Click on it and observe the blue border

[demo](https://codesandbox.io/s/b-dropdown-item-button-styling-1ier8?file=/App.vue)

### Expected behavior

I would expect the designs of these to behave consistently on all browsers, either both with or both without the `outline` (preferably without).

### Versions

**Libraries:**

- BootstrapVue: 2.16.0
- Bootstrap: 4.5.3
- Vue: 2.6.12

**Environment:**

- Device: Mac
- OS: Mojave
- Browser: Firefox, Chromium
- Version: 81.02, 86.0.4240.99

### Demo link

https://codesandbox.io/s/b-dropdown-item-button-styling-1ier8?file=/App.vue

**Comment by Hiws** [comment]
This behavior looks to be is the same as with native Bootstrap.
<https://getbootstrap.com/docs/4.5/components/dropdowns/#menu-items>
<https://v5.getbootstrap.com/docs/5.0/components/dropdowns/#menu-items>

So I don't think we should change this behavior here, as this is more of an upstream issue.
You could try suggesting it on Bootstrap's GitHub.
<https://github.com/twbs/bootstrap/issues>

**Comment by bbugh** [comment]
Looks like you're right. I thought I checked it in upstream bootstrap, but it's not impossible that while juggling a bunch of browsers I looked at it in Firefox by accident and thought it was fine.

It looks like @tmorehouse has brought this same issue up a few times on bootstrap's repo. https://github.com/twbs/bootstrap/issues/23329#issuecomment-331238806 and https://github.com/twbs/bootstrap/pull/23990#issuecomment-333345957.

I will make a new issue over there.
